app/helpers/rails_admin/application_helper.rb

```diff
@@ -94,7 +94,7 @@ def root_navigation
       actions(:root).select(&:show_in_sidebar).group_by(&:sidebar_label).collect do |label, nodes|
         li_stack = nodes.map do |node|
           url = rails_admin.url_for(action: node.action_name, controller: 'rails_admin/main')
-          nav_icon = node.link_icon ? %(<i class="#{node.link_icon}"></i>).html_safe : ''
+          nav_icon = node.link_icon ? %(<i class="#{node.link_icon}"></i> ).html_safe : ''
```

Comment:
We don't need this, since we have `link_to nav_icon + " " + wording_for...` just below this.